packages/vendure-plugin-invoices/src/api/default-template.ts

```diff
@@ -1,113 +1,234 @@
 export const defaultTemplate = `
+
 <!DOCTYPE html>
-<html style="zoom: 0.55">
+<html style="margin: 0;">
+
 <head>
-  <meta charset="utf-8" />
-  <title>Order {{ order.code }}</title>
+    <meta charset="utf-8" />
+        <style>
+            @import url('https://fonts.googleapis.com/css2?family=IBM+Plex+Mono:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;1,100;1,200;1,300;1,400;1,500;1,600;1,700&display=swap');
+
+            h1 {
+                font-size: 1.9rem;
+                margin: 0.5rem;
+            }
+            h6 {
+                font-size: 0.6rem;
+                margin: 0;
+            }
+            h4 {
+                font-size: 0.8rem;
+            }
+            h5 {
+                font-size: 0.8rem;
+                margin: 0;
+            }
+            h3 {
+                font-size: 1.2rem;
+            }
+            h20 {
+                font-size: 0.8rem;
+                color: #00000080;
+                font-weight: normal;
+            }
+
+            #quantity-info {
+                width: 10%;
+            }
+
+            #product-info {
+                width: 58%;
+            }
+
+            #vat-info {
+                width: 20%;
+            }
+
+            #subtotal-info {
+                width: 20%;
+            }
```

Comment:
Is er een reden waarom je een css id selector hebt gebruikt inplaats van een class? (waarom `#vat-info` ipv `.vat-info`)

In een for-each loop worden de order regels meerdere keren ge-renderd, waardoor er meerdere regels zijn met eenzelfde `id="quantity-info"`, en dat is geen valide HTML.